mcs/tools/corcompare/AssemblyResolver.cs

```diff
@@ -32,9 +32,13 @@
 
 using Mono.Cecil;
 
-namespace GuiCompare {
+#if USE_MONO_API_TOOLS_NAMESPACE
+namespace Mono.ApiTools {
+#else
+namespace CorCompare {
```

Comment:
There appears to be a weird namespace difference here. I changed it to match the rest, but I can revert this.

Reply:
I think we could change the namespace to `Mono.ApiTools` everywhere and remove the define, the .exe's shouldn't be loaded via reflection so nobody should be broken by the namespace change.